src/application/dataFetcher/random/RandomSgxDataFetcherAdapter.ts

```diff
@@ -20,8 +22,8 @@ class RandomSgxDataFetcherAdapter implements DataFetcher {
   }
 
   async fetch(request: Request) {
-    const min = RequestUrlParser.resolveLeftSideBound(request.url);
-    const max = RequestUrlParser.resolveRightSideBound(request.url);
+    const min: number = +RequestUrlParser.resolveLeftSideBound(request.url);
```

Comment:
TS should know that min is number

Reply:
do you mean like this? `const min = +RequestUrlParser.resolveLeftSideBound(request.url);`